django/core/management/commands/inspectdb.py

```diff
@@ -19,6 +19,8 @@ def add_arguments(self, parser):
         parser.add_argument('--database', action='store', dest='database',
             default=DEFAULT_DB_ALIAS, help='Nominates a database to '
             'introspect. Defaults to using the "default" database.')
+        parser.add_argument('table', action='store', nargs='*', type=str,
```

Comment:
Is there a justification for not using the regular `--` prefix for argument names?